cluster-provision/gocli/docker/docker.go

```diff
@@ -43,6 +44,26 @@ func GetPrefixedVolumes(cli *client.Client, prefix string) ([]*types.Volume, err
 
 func ImagePull(cli *client.Client, ctx context.Context, ref string, options types.ImagePullOptions) error {
 
+	if !strings.ContainsAny(ref, ":@") {
+		ref = ref + ":latest"
+	}
+	ref = strings.TrimPrefix(ref, "docker.io/")
```

Comment:
I remember there were some changes on quota at docker.io and we may need to move to quay.io let's at least move `docker.io` to a constant.

Reply:
This is only here because docker.io images are treated special on the client side. The `docker.io` prefix is basically not shown when listing the images.

Reply:
Got it so this is compatible with quay.io.